File: app/src/main/java/com/forkmang/adapter/SpinnnerAdapter_Floor.java
package com.forkmang.adapter;

import android.annotation.SuppressLint;
import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.BaseAdapter;
import android.widget.TextView;

import com.forkmang.R;
import com.forkmang.data.FlooDropdown;

import java.util.ArrayList;

public class SpinnnerAdapter_Floor extends BaseAdapter {
    Context context;
    ArrayList<FlooDropdown> flooDropdownArrayList;
    LayoutInflater inflter;

    public SpinnnerAdapter_Floor(Context applicationContext, ArrayList<FlooDropdown> flooDropdownArrayList) {
        this.context = applicationContext;
        this.flooDropdownArrayList = flooDropdownArrayList;
        inflter = (LayoutInflater.from(applicationContext));
    }

    @Override
    public int getCount() {
        return flooDropdownArrayList.size();
    }

    @Override
    public Object getItem(int position) {
        return null;
    }

    @Override
    public long getItemId(int position) {
        return 0;
    }

    @Override
    public View getView(int position, View convertView, ViewGroup parent)
    {
        @SuppressLint({"ViewHolder", "InflateParams"})
        View view = inflter.inflate(R.layout.spinner_cell, null);
        TextView person = view.findViewById(R.id.txt_person);
        FlooDropdown flooDropdown = flooDropdownArrayList.get(position);
        person.setText(flooDropdown.getFloor_name());
        return view;
    }
}

File: app/src/main/java/com/forkmang/adapter/SpinnnerAdapterFloor.kt
package com.forkmang.adapter

import android.annotation.SuppressLint
import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.BaseAdapter
import android.widget.TextView
import com.forkmang.R
import com.forkmang.data.FlooDropdown

class SpinnnerAdapterFloor(
    var context: Context,
    var flooDropdownArrayList: ArrayList<FlooDropdown>
) : BaseAdapter() {
    var inflter: LayoutInflater = LayoutInflater.from(context)

    override fun getCount(): Int {
        return flooDropdownArrayList.size
    }

    override fun getItem(position: Int): Any {
        return flooDropdownArrayList[position]
    }

    override fun getItemId(position: Int): Long {
        return 0
    }

    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        @SuppressLint("ViewHolder", "InflateParams") val view =
            inflter.inflate(R.layout.spinner_cell, null)
        val person = view.findViewById<TextView>(R.id.txt_person)
        val flooDropdown = flooDropdownArrayList[position]
        person.text = flooDropdown.floor_name
        return view
    }
}